### ISSUE TITLE
Cannot start Mapbox Studio version Darwn x64-v0.1.1

### ISSUE THREAD

**Issue by savannahjune** [mapbox/mapbox-studio-classic#837]
I cannot start Mapbox Studio version Darwn x64-v0.1.1 either from downloading it from https://www.mapbox.com/mapbox-studio/#darwin or by cloning it from github.

When I download via github I get this error from npm-debug.log:

0 info it worked if it ends with ok
1 verbose cli [ 'node', '/usr/local/bin/npm', 'start' ]
2 info using npm@2.0.0-alpha-5
3 info using node@v0.10.31
4 verbose node symlink /usr/local/bin/node
5 verbose run-script [ 'prestart', 'start', 'poststart' ]
6 info prestart mapbox-studio@0.1.1
7 info start mapbox-studio@0.1.1
8 verbose unsafe-perm in lifecycle true
9 info mapbox-studio@0.1.1 Failed to exec start script
10 error mapbox-studio@0.1.1 start: `./index.js`
10 error Exit status 8
11 error Failed at the mapbox-studio@0.1.1 start script.
11 error This is most likely a problem with the mapbox-studio package,
11 error not with npm itself.
11 error Tell the author that this fails on your system:
11 error     ./index.js
11 error You can get their info via:
11 error     npm owner ls mapbox-studio
11 error There is likely additional logging output above.
12 error System Darwin 13.3.0
13 error command "node" "/usr/local/bin/npm" "start"
14 error cwd /Users/Savannah/mapbox-studio
15 error node -v v0.10.31
16 error npm -v 2.0.0-alpha-5
17 error code ELIFECYCLE
18 verbose exit [ 1, true ]

When I download it from mapbox.com I get this error from the console: 

![failedtolaunch](https://cloud.githubusercontent.com/assets/2073292/4257740/6d2b7e6a-3ac7-11e4-828e-19598aad17d9.png)

Thanks!

**Comment by yhahn** [comment]
Hi @savannahjune -- we're having trouble reproducing this issue which always makes it tougher to debug : ) Let's start by trying to get your `git clone` working -- could you post the output of the command

```
npm ls
```

From the directory you cloned mapbox-studio to?

**Comment by savannahjune** [comment]
Hi @yhahn ,

Sure thing, here's the output from the directory:

```
mapbox-studio@0.1.1 /Users/Savannah/mapbox-studio
├── abaculus@0.0.5
├─┬ carto@0.13.0
│ ├── mapnik-reference@5.1.0
│ ├─┬ optimist@0.6.1
│ │ ├── minimist@0.0.10
│ │ └── wordwrap@0.0.2
│ └── underscore@1.6.0
├── cors@2.3.2
├── dirty@0.9.9
├─┬ express@3.1.2
│ ├── buffer-crc32@0.2.3
│ ├── commander@0.6.1
│ ├─┬ connect@2.7.5
│ │ ├── buffer-crc32@0.1.1
│ │ ├── bytes@0.2.0
│ │ ├── formidable@1.0.11
│ │ ├── pause@0.0.1
│ │ └── qs@0.5.1
│ ├── cookie@0.0.5
│ ├── cookie-signature@1.0.0
│ ├─┬ debug@2.0.0
│ │ └── ms@0.6.2
│ ├── fresh@0.1.0
│ ├── methods@0.0.1
│ ├── mkdirp@0.3.5
│ ├── range-parser@0.0.4
│ └─┬ send@0.1.0
│   └── mime@1.2.6
├─┬ fstream@0.1.31
│ ├── graceful-fs@3.0.2
│ └── inherits@2.0.1
├── gazetteer@0.1.2
├── getport@0.1.0
├─┬ js-yaml@2.1.0
│ ├─┬ argparse@0.1.15
│ │ └── underscore.string@2.3.3
│ └── esprima@1.0.4
├── log-rotate@0.2.7
├── mapbox-studio-comic@1.0.1
├── mapbox-studio-default-fonts@0.0.4
├── mapbox-studio-default-source@1.0.0
├── mapbox-studio-default-style@1.0.0
├── mapbox-studio-highcontrast@1.0.6
├── mapbox-studio-light@1.0.1
├── mapbox-studio-looseleaf@1.0.1
├── mapbox-studio-mapbox-outdoors@1.0.1
├── mapbox-studio-osm-bright@1.0.2
├── mapbox-studio-pencil@1.0.1
├── mapbox-studio-pirates@1.0.0
├── mapbox-studio-pro-fonts@1.0.0
├── mapbox-studio-run-bike-and-hike@1.0.2
├── mapbox-studio-satellite-afternoon@1.0.1
├── mapbox-studio-wheatpaste@1.0.1
├─┬ mapbox-upload@1.1.1
│ ├─┬ aws-sdk@1.18.0
│ │ ├─┬ xml2js@0.2.4
│ │ │ └── sax@0.6.0
│ │ └── xmlbuilder@0.4.2
│ ├─┬ request@2.37.0
│ │ ├── aws-sign2@0.5.0
│ │ ├── forever-agent@0.5.2
│ │ ├─┬ form-data@0.1.4
│ │ │ ├── async@0.9.0
│ │ │ ├─┬ combined-stream@0.0.5
│ │ │ │ └── delayed-stream@0.0.5
│ │ │ └── mime@1.2.11
│ │ ├─┬ hawk@1.1.1
│ │ │ ├── boom@0.4.2
│ │ │ ├── cryptiles@0.2.2
│ │ │ ├── hoek@0.9.1
│ │ │ └── sntp@0.2.4
│ │ ├─┬ http-signature@0.10.0
│ │ │ ├── asn1@0.1.11
│ │ │ ├── assert-plus@0.1.2
│ │ │ └── ctype@0.5.2
│ │ ├── json-stringify-safe@5.0.0
│ │ ├── mime-types@1.0.2
│ │ ├── node-uuid@1.4.1
│ │ ├── oauth-sign@0.3.0
│ │ ├── qs@0.6.6
│ │ ├─┬ tough-cookie@0.12.1
│ │ │ └── punycode@1.3.1
│ │ └── tunnel-agent@0.4.0
│ └─┬ s3-upload-stream@0.6.2
│   └─┬ aws-sdk@2.0.15
│     ├── aws-sdk-apis@3.1.8
│     ├─┬ xml2js@0.2.6
│     │ └── sax@0.4.2
│     └── xmlbuilder@0.4.2
├─┬ mapnik@1.4.15
│ ├── mapnik-vector-tile@0.5.5
│ ├── nan@1.2.0
│ └─┬ node-pre-gyp@0.5.25
│   ├─┬ mkdirp@0.5.0
│   │ └── minimist@0.0.8
│   ├─┬ nopt@3.0.1
│   │ └── abbrev@1.0.5
│   ├─┬ npmlog@0.1.1
│   │ └── ansi@0.3.0
│   ├─┬ rc@0.5.1
│   │ ├── deep-extend@0.2.11
│   │ ├── ini@1.1.0
│   │ ├── minimist@0.0.10
│   │ └── strip-json-comments@0.1.3
│   ├─┬ request@2.40.0
│   │ ├── aws-sign2@0.5.0
│   │ ├── forever-agent@0.5.2
│   │ ├─┬ form-data@0.1.4
│   │ │ ├── async@0.9.0
│   │ │ ├─┬ combined-stream@0.0.5
│   │ │ │ └── delayed-stream@0.0.5
│   │ │ └── mime@1.2.11
│   │ ├─┬ hawk@1.1.1
│   │ │ ├── boom@0.4.2
│   │ │ ├── cryptiles@0.2.2
│   │ │ ├── hoek@0.9.1
│   │ │ └── sntp@0.2.4
│   │ ├─┬ http-signature@0.10.0
│   │ │ ├── asn1@0.1.11
│   │ │ ├── assert-plus@0.1.2
│   │ │ └── ctype@0.5.2
│   │ ├── json-stringify-safe@5.0.0
│   │ ├── mime-types@1.0.2
│   │ ├── node-uuid@1.4.1
│   │ ├── oauth-sign@0.3.0
│   │ ├── qs@1.0.2
│   │ ├── stringstream@0.0.4
│   │ ├─┬ tough-cookie@0.12.1
│   │ │ └── punycode@1.3.1
│   │ └── tunnel-agent@0.4.0
│   ├── rimraf@2.2.8
│   ├── semver@3.0.1
│   ├─┬ tar@1.0.1
│   │ ├── block-stream@0.0.7
│   │ ├─┬ fstream@1.0.2
│   │ │ └── graceful-fs@3.0.2
│   │ └── inherits@2.0.1
│   └─┬ tar-pack@2.0.0
│     ├── debug@0.7.4
│     ├─┬ fstream@0.1.31
│     │ ├── graceful-fs@3.0.2
│     │ └── inherits@2.0.1
│     ├─┬ fstream-ignore@0.0.7
│     │ ├── inherits@2.0.1
│     │ └─┬ minimatch@0.2.14
│     │   ├── lru-cache@2.5.0
│     │   └── sigmund@1.0.0
│     ├── graceful-fs@1.2.3
│     ├── once@1.1.1
│     ├─┬ readable-stream@1.0.31
│     │ ├── core-util-is@1.0.1
│     │ ├── inherits@2.0.1
│     │ ├── isarray@0.0.1
│     │ └── string_decoder@0.10.31
│     ├─┬ tar@0.1.20
│     │ ├── block-stream@0.0.7
│     │ └── inherits@2.0.1
│     └── uid-number@0.0.3
├─┬ mapnik-omnivore@1.2.12
│ ├─┬ gdal@0.1.2
│ │ └─┬ node-pre-gyp@0.5.19
│ │   ├─┬ mkdirp@0.5.0
│ │   │ └── minimist@0.0.8
│ │   ├─┬ nopt@2.2.1
│ │   │ └── abbrev@1.0.5
│ │   ├─┬ npmlog@0.0.6
│ │   │ └── ansi@0.2.1
│ │   ├─┬ rc@0.4.0
│ │   │ ├── deep-extend@0.2.10
│ │   │ ├── ini@1.1.0
│ │   │ ├── minimist@0.0.10
│ │   │ └── strip-json-comments@0.1.3
│ │   ├─┬ request@2.36.0
│ │   │ ├── aws-sign2@0.5.0
│ │   │ ├── forever-agent@0.5.2
│ │   │ ├─┬ form-data@0.1.2
│ │   │ │ ├── async@0.2.10
│ │   │ │ └─┬ combined-stream@0.0.4
│ │   │ │   └── delayed-stream@0.0.5
│ │   │ ├─┬ hawk@1.0.0
│ │   │ │ ├── boom@0.4.2
│ │   │ │ ├── cryptiles@0.2.2
│ │   │ │ ├── hoek@0.9.1
│ │   │ │ └── sntp@0.2.4
│ │   │ ├─┬ http-signature@0.10.0
│ │   │ │ ├── asn1@0.1.11
│ │   │ │ ├── assert-plus@0.1.2
│ │   │ │ └── ctype@0.5.2
│ │   │ ├── json-stringify-safe@5.0.0
│ │   │ ├── mime@1.2.11
│ │   │ ├── node-uuid@1.4.1
│ │   │ ├── oauth-sign@0.3.0
│ │   │ ├── qs@0.6.6
│ │   │ ├─┬ tough-cookie@0.12.1
│ │   │ │ └── punycode@1.2.4
│ │   │ └── tunnel-agent@0.4.0
│ │   ├── rimraf@2.2.8
│ │   ├── semver@2.3.0
│ │   ├─┬ tar@0.1.19
│ │   │ ├── block-stream@0.0.7
│ │   │ ├─┬ fstream@0.1.25
│ │   │ │ ├── graceful-fs@2.0.3
│ │   │ │ └── mkdirp@0.3.5
│ │   │ └── inherits@2.0.1
│ │   └─┬ tar-pack@2.0.0
│ │     ├── debug@0.7.4
│ │     ├─┬ fstream@0.1.25
│ │     │ ├── graceful-fs@2.0.3
│ │     │ ├── inherits@2.0.1
│ │     │ └── mkdirp@0.3.5
│ │     ├─┬ fstream-ignore@0.0.7
│ │     │ ├── inherits@2.0.1
│ │     │ └─┬ minimatch@0.2.14
│ │     │   ├── lru-cache@2.5.0
│ │     │   └── sigmund@1.0.0
│ │     ├── graceful-fs@1.2.3
│ │     ├── once@1.1.1
│ │     ├─┬ readable-stream@1.0.27-1
│ │     │ ├── core-util-is@1.0.1
│ │     │ ├── inherits@2.0.1
│ │     │ ├── isarray@0.0.1
│ │     │ └── string_decoder@0.10.25-1
│ │     └── uid-number@0.0.3
│ └─┬ srs@0.4.3
│   ├── nan@1.2.0
│   └─┬ node-pre-gyp@0.5.17
│     ├─┬ mkdirp@0.5.0
│     │ └── minimist@0.0.8
│     ├─┬ nopt@2.2.1
│     │ └── abbrev@1.0.5
│     ├─┬ npmlog@0.0.6
│     │ └── ansi@0.2.1
│     ├─┬ rc@0.4.0
│     │ ├── deep-extend@0.2.10
│     │ ├── ini@1.1.0
│     │ ├── minimist@0.0.10
│     │ └── strip-json-comments@0.1.3
│     ├─┬ request@2.36.0
│     │ ├── aws-sign2@0.5.0
│     │ ├── forever-agent@0.5.2
│     │ ├─┬ form-data@0.1.2
│     │ │ ├── async@0.2.10
│     │ │ └─┬ combined-stream@0.0.4
│     │ │   └── delayed-stream@0.0.5
│     │ ├─┬ hawk@1.0.0
│     │ │ ├── boom@0.4.2
│     │ │ ├── cryptiles@0.2.2
│     │ │ ├── hoek@0.9.1
│     │ │ └── sntp@0.2.4
│     │ ├─┬ http-signature@0.10.0
│     │ │ ├── asn1@0.1.11
│     │ │ ├── assert-plus@0.1.2
│     │ │ └── ctype@0.5.2
│     │ ├── json-stringify-safe@5.0.0
│     │ ├── mime@1.2.11
│     │ ├── node-uuid@1.4.1
│     │ ├── oauth-sign@0.3.0
│     │ ├── qs@0.6.6
│     │ ├─┬ tough-cookie@0.12.1
│     │ │ └── punycode@1.2.4
│     │ └── tunnel-agent@0.4.0
│     ├── rimraf@2.2.8
│     ├── semver@2.3.0
│     ├─┬ tar@0.1.19
│     │ ├── block-stream@0.0.7
│     │ ├─┬ fstream@0.1.25
│     │ │ ├── graceful-fs@2.0.3
│     │ │ └── mkdirp@0.3.5
│     │ └── inherits@2.0.1
│     └─┬ tar-pack@2.0.0
│       ├── debug@0.7.4
│       ├─┬ fstream@0.1.25
│       │ ├── graceful-fs@2.0.3
│       │ ├── inherits@2.0.1
│       │ └── mkdirp@0.3.5
│       ├─┬ fstream-ignore@0.0.7
│       │ ├── inherits@2.0.1
│       │ └─┬ minimatch@0.2.14
│       │   ├── lru-cache@2.5.0
│       │   └── sigmund@1.0.0
│       ├── graceful-fs@1.2.3
│       ├── once@1.1.1
│       ├─┬ readable-stream@1.0.27-1
│       │ ├── core-util-is@1.0.1
│       │ ├── inherits@2.0.1
│       │ ├── isarray@0.0.1
│       │ └── string_decoder@0.10.25-1
│       └── uid-number@0.0.3
├── mapnik-reference@5.0.9
├── mapnik-test-data@2.0.1
├─┬ mbtiles@0.5.0
│ ├─┬ sqlite3@2.2.7
│ │ ├── nan@1.1.2
│ │ ├─┬ node-pre-gyp@0.5.22
│ │ │ ├─┬ mkdirp@0.5.0
│ │ │ │ └── minimist@0.0.8
│ │ │ ├─┬ nopt@3.0.1
│ │ │ │ └── abbrev@1.0.5
│ │ │ ├─┬ npmlog@0.1.1
│ │ │ │ └── ansi@0.3.0
│ │ │ ├─┬ rc@0.5.0
│ │ │ │ ├── deep-extend@0.2.11
│ │ │ │ ├── ini@1.1.0
│ │ │ │ ├── minimist@0.0.10
│ │ │ │ └── strip-json-comments@0.1.3
│ │ │ ├─┬ request@2.39.0
│ │ │ │ ├── aws-sign2@0.5.0
│ │ │ │ ├── forever-agent@0.5.2
│ │ │ │ ├─┬ form-data@0.1.4
│ │ │ │ │ ├── async@0.9.0
│ │ │ │ │ ├─┬ combined-stream@0.0.5
│ │ │ │ │ │ └── delayed-stream@0.0.5
│ │ │ │ │ └── mime@1.2.11
│ │ │ │ ├─┬ hawk@1.1.1
│ │ │ │ │ ├── boom@0.4.2
│ │ │ │ │ ├── cryptiles@0.2.2
│ │ │ │ │ ├── hoek@0.9.1
│ │ │ │ │ └── sntp@0.2.4
│ │ │ │ ├─┬ http-signature@0.10.0
│ │ │ │ │ ├── asn1@0.1.11
│ │ │ │ │ ├── assert-plus@0.1.2
│ │ │ │ │ └── ctype@0.5.2
│ │ │ │ ├── json-stringify-safe@5.0.0
│ │ │ │ ├── mime-types@1.0.2
│ │ │ │ ├── node-uuid@1.4.1
│ │ │ │ ├── oauth-sign@0.3.0
│ │ │ │ ├── qs@0.6.6
│ │ │ │ ├── stringstream@0.0.4
│ │ │ │ ├─┬ tough-cookie@0.12.1
│ │ │ │ │ └── punycode@1.3.0
│ │ │ │ └── tunnel-agent@0.4.0
│ │ │ ├── rimraf@2.2.8
│ │ │ ├── semver@3.0.1
│ │ │ ├─┬ tar@1.0.0
│ │ │ │ ├── block-stream@0.0.7
│ │ │ │ ├─┬ fstream@1.0.1
│ │ │ │ │ └── graceful-fs@3.0.2
│ │ │ │ └── inherits@2.0.1
│ │ │ └─┬ tar-pack@2.0.0
│ │ │   ├── debug@0.7.4
│ │ │   ├─┬ fstream@0.1.31
│ │ │   │ ├── graceful-fs@3.0.2
│ │ │   │ └── inherits@2.0.1
│ │ │   ├─┬ fstream-ignore@0.0.7
│ │ │   │ ├── inherits@2.0.1
│ │ │   │ └─┬ minimatch@0.2.14
│ │ │   │   ├── lru-cache@2.5.0
│ │ │   │   └── sigmund@1.0.0
│ │ │   ├── graceful-fs@1.2.3
│ │ │   ├── once@1.1.1
│ │ │   ├─┬ readable-stream@1.0.27-1
│ │ │   │ ├── core-util-is@1.0.1
│ │ │   │ ├── inherits@2.0.1
│ │ │   │ ├── isarray@0.0.1
│ │ │   │ └── string_decoder@0.10.25-1
│ │ │   ├─┬ tar@0.1.20
│ │ │   │ ├── block-stream@0.0.7
│ │ │   │ └── inherits@2.0.1
│ │ │   └── uid-number@0.0.3
│ │ └── set-immediate@0.1.1
│ └── tiletype@0.1.0
├── minimist@0.1.0
├─┬ mkdirp@0.5.0
│ └── minimist@0.0.8
├─┬ passport@0.2.1
│ ├── passport-strategy@1.0.0
│ └── pause@0.0.1
├─┬ passport-oauth@1.0.0
│ ├─┬ passport-oauth1@1.0.1
│ │ ├── oauth@0.9.12
│ │ ├── passport-strategy@1.0.0
│ │ └── utils-merge@1.0.0
│ └─┬ passport-oauth2@1.1.2
│   ├── oauth@0.9.12
│   ├── passport-strategy@1.0.0
│   └── uid2@0.0.3
├─┬ phantomjs@1.9.7-15
│ ├── adm-zip@0.2.1
│ ├── kew@0.1.7
│ ├── mkdirp@0.3.5
│ ├── ncp@0.4.2
│ ├─┬ npmconf@0.0.24
│ │ ├─┬ config-chain@1.1.8
│ │ │ └── proto-list@1.2.3
│ │ ├── inherits@1.0.0
│ │ ├── ini@1.1.0
│ │ ├─┬ nopt@2.2.1
│ │ │ └── abbrev@1.0.5
│ │ ├── once@1.1.1
│ │ ├── osenv@0.0.3
│ │ └── semver@1.1.4
│ ├── progress@1.1.8
│ ├─┬ request-progress@0.3.1
│ │ └── throttleit@0.0.2
│ └── which@1.0.5
├─┬ progress-stream@0.5.0
│ ├── single-line-log@0.3.1
│ └─┬ through2@0.2.3
│   ├─┬ readable-stream@1.1.13
│   │ ├── core-util-is@1.0.1
│   │ ├── inherits@2.0.1
│   │ ├── isarray@0.0.1
│   │ └── string_decoder@0.10.31
│   └─┬ xtend@2.1.2
│     └── object-keys@0.4.0
├── queue-async@1.0.7
├─┬ request@2.36.0
│ ├── aws-sign2@0.5.0
│ ├── forever-agent@0.5.2
│ ├─┬ form-data@0.1.4
│ │ ├── async@0.9.0
│ │ └─┬ combined-stream@0.0.5
│ │   └── delayed-stream@0.0.5
│ ├─┬ hawk@1.0.0
│ │ ├── boom@0.4.2
│ │ ├── cryptiles@0.2.2
│ │ ├── hoek@0.9.1
│ │ └── sntp@0.2.4
│ ├─┬ http-signature@0.10.0
│ │ ├── asn1@0.1.11
│ │ ├── assert-plus@0.1.2
│ │ └── ctype@0.5.2
│ ├── json-stringify-safe@5.0.0
│ ├── mime@1.2.11
│ ├── node-uuid@1.4.1
│ ├── oauth-sign@0.3.0
│ ├── qs@0.6.6
│ ├─┬ tough-cookie@0.12.1
│ │ └── punycode@1.3.1
│ └── tunnel-agent@0.4.0
├── rimraf@2.2.8
├── safer-stringify@0.0.1
├── speedometer@0.1.2
├── sphericalmercator@1.0.2
├─┬ tape@2.13.4
│ ├── deep-equal@0.2.1
│ ├── defined@0.0.0
│ ├─┬ glob@3.2.11
│ │ └─┬ minimatch@0.3.0
│ │   ├── lru-cache@2.5.0
│ │   └── sigmund@1.0.0
│ ├── inherits@2.0.1
│ ├── object-inspect@0.4.0
│ └── resumer@0.0.0
├─┬ tar@0.1.20
│ ├── block-stream@0.0.7
│ └── inherits@2.0.1
├── through@2.3.4
├─┬ tilejson@0.10.0
│ ├── agentkeepalive@0.1.5
│ ├── get@1.3.0
│ └── tiletype@0.1.0
├─┬ tilelive@5.2.1
│ └── minimist@0.2.0
├─┬ tilelive-bridge@0.6.0
│ └─┬ mapnik-pool@0.0.0
│   ├── generic-pool@2.0.4
│   └── xtend@3.0.0
├─┬ tilelive-vector@2.1.0
│ ├── numeral@1.5.3
│ ├─┬ request@2.27.0
│ │ ├── aws-sign@0.3.0
│ │ ├── cookie-jar@0.3.0
│ │ ├── forever-agent@0.5.2
│ │ ├─┬ form-data@0.1.4
│ │ │ ├── async@0.9.0
│ │ │ └─┬ combined-stream@0.0.5
│ │ │   └── delayed-stream@0.0.5
│ │ ├─┬ hawk@1.0.0
│ │ │ ├── boom@0.4.2
│ │ │ ├── cryptiles@0.2.2
│ │ │ ├── hoek@0.9.1
│ │ │ └── sntp@0.2.4
│ │ ├─┬ http-signature@0.10.0
│ │ │ ├── asn1@0.1.11
│ │ │ ├── assert-plus@0.1.2
│ │ │ └── ctype@0.5.2
│ │ ├── json-stringify-safe@5.0.0
│ │ ├── mime@1.2.11
│ │ ├── node-uuid@1.4.1
│ │ ├── oauth-sign@0.3.0
│ │ ├── qs@0.6.6
│ │ └── tunnel-agent@0.3.0
│ ├─┬ spherical@0.0.1
│ │ └── wgs84@0.0.0
│ ├─┬ tilelive@5.0.2
│ │ └── minimist@0.2.0
│ └── underscore@1.6.0
└── underscore@1.4.4 
```

Thanks,
Savannah

**Comment by ianshward** [comment]
@savannahjune could you try installing an older version of NPM, like 1.4.27, remove the node_modules folder, and do a fresh `npm install`?  This is somewhat of a shot in the dark, based on unexpected behavior I saw today using NPM 2.0.0 and node 0.10.31, but with a different node project.

**Comment by savannahjune** [comment]
switched npm versions. latest error:

npm ERR! Error: ENOENT, open '/Users/Savannah/mapbox-studio/node_modules/package.json'
npm ERR! If you need help, you may report this _entire_ log,
npm ERR! including the npm and node versions, at:
npm ERR!     http://github.com/npm/npm/issues

npm ERR! System Darwin 13.3.0
npm ERR! command "node" "/usr/local/bin/npm" "start"
npm ERR! cwd /Users/Savannah/mapbox-studio/node_modules
npm ERR! node -v v0.10.31
npm ERR! npm -v 1.4.27
npm ERR! path /Users/Savannah/mapbox-studio/node_modules/package.json
npm ERR! code ENOENT
npm ERR! errno 34
npm ERR! 
npm ERR! Additional logging details can be found in:
npm ERR!     /Users/Savannah/mapbox-studio/node_modules/npm-debug.log
npm ERR! not ok code 0

from npm-debug.log:

```
17 error owner ls Couldn't get owner data mapbox-studio
18 error 404 404 Not Found: mapbox-studio
18 error 404
18 error 404 'mapbox-studio' is not in the npm registry.
18 error 404 You should bug the author to publish it
18 error 404
18 error 404 Note that you can also install from a
18 error 404 tarball, folder, or http url, or git url.
19 error System Darwin 13.3.0
20 error command "node" "/usr/local/bin/npm" "owner" "ls" "mapbox-studio"
21 error cwd /Users/Savannah/mapbox-studio
22 error node -v v0.10.31
23 error npm -v 1.4.27
24 error code E404
25 verbose exit [ 1, true ]
```

Thanks!

**Comment by yhahn** [comment]
^^ @savannahjune it looks like you might have run `npm install` from the `mapbox-studio/node_modules` directory. You'll want to cd back to the `mapbox-studio` directory and from there:

```
rm -rf node_modules
npm install
```

**Comment by savannahjune** [comment]
I cd'ed back into mapbox-studio, ran rm-rf node_modules, and then npm install.  

```
savannahs-air:~ Savannah$ cd mapbox-studio/
savannahs-air:mapbox-studio Savannah$ rm -rf node_modules
savannahs-air:mapbox-studio Savannah$ npm install
```

Install worked, with only one warning:

```
[mapnik] Success: "/Users/Savannah/mapbox-studio/node_modules/mapnik/lib/binding/node-v11-darwin-x64/mapnik.node" is installed via remote
npm WARN deprecated set-immediate@0.1.1: Use `setimmediate` instead
```

Then I tried npm start and got this error message:

```

npm ERR! mapbox-studio@0.1.2 start: `./index.js`
npm ERR! Exit status 8
npm ERR! 
npm ERR! Failed at the mapbox-studio@0.1.2 start script.
npm ERR! This is most likely a problem with the mapbox-studio package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     ./index.js
npm ERR! You can get their info via:
npm ERR!     npm owner ls mapbox-studio
npm ERR! There is likely additional logging output above.
npm ERR! System Darwin 13.3.0
npm ERR! command "node" "/usr/local/bin/npm" "start"
npm ERR! cwd /Users/Savannah/mapbox-studio
npm ERR! node -v v0.10.31
npm ERR! npm -v 1.4.27
npm ERR! code ELIFECYCLE
npm ERR! 
npm ERR! Additional logging details can be found in:
npm ERR!     /Users/Savannah/mapbox-studio/npm-debug.log
npm ERR! not ok code 0
```

**Comment by yhahn** [comment]
@savannahjune if you run

```
node index.js
```

Do you get the same error?

**Comment by savannahjune** [comment]
I get this error:

```
savannahs-air:mapbox-studio Savannah$ node index.js

events.js:72
        throw er; // Unhandled 'error' event
              ^
Error: EACCES, open '/Users/Savannah/.mapbox-studio/app.db'
```

**Comment by yhahn** [comment]
@savannahjune hrm, this likely means that `.mapbox-studio/app.db` and other files in that directory have permissions that your current user doesn't have access to write to.

Did you ever run mapbox studio as root/using `sudo` or as an Administrator?

**Comment by savannahjune** [comment]
Hi @yhahn ,

Great news, studio's working for me again!  I do not think I ever ran mapbox studio as root/using sudo or as an Administrator but I did take a stab at re-downloading the files after deleting the .mapbox-studio folder from users/Savannah where the main mapbox-studio folder is located. 

Then it worked properly and I am now using studio again!  Not sure if it was because I deleted and re-downloaded without using as root/using sudo or as an Administartor or if it was because I deleted the .mapbox-studio folder. I didn't notice the folder until the error mentioned the .mapbox-studio/app.db file, so this is the first time I've tried that.

Thanks so much for helping me!  Thanks also to @geografa too who helped me last night.
